tools/filters/catWrapper.xml

```diff
@@ -1,13 +1,13 @@
 <tool id="cat1" name="Concatenate datasets" version="1.0.0">
     <description>tail-to-head</description>
-    <command interpreter="python">
-        catWrapper.py
-        $out_file1
-        $input1
-        #for $q in $queries
-            ${q.input2}
-        #end for
-    </command>
+    <command interpreter="python"><![CDATA[
```

Comment:
I think the command interpreter part doesn't quote the path.

Reply:
Yep, fixed that in the next commit!